tests/console/validate_ext4_fs.pm

```diff
@@ -0,0 +1,66 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Validate ext4 filesystem and system partitioning
+# Maintainer: Sofia Syrianidou <ssyrianidou@suse.com>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use scheduler 'get_test_suite_data';
+use Test::Assert ':all';
+
+
+sub run {
+
+    select_console "root-console";
+
+#Validate that swap is created and /home is on seperate partition. 
+script_run("df -PH /");
+script_run("df -PH /home");
+save_screenshot;
+
+    assert_script_run("lsblk -n | grep 'SWAP'", fail_message => "/swap partition was not created.");
+    assert_script_run("lsblk -n | grep 'home'", fail_message => "/home is not on separate partition.");
+
+#Validate that the filesystem type is the expected one.
+ 
+    my $test_data  = get_test_suite_data();
+    my @partitions = @{$test_data->{partitions}};
+
+    foreach (@partitions) {
+
+#Validate that the filesystem type is the expected one.
+
+        record_info("Check fs $_->{mnt_point}", "Verify the '$_->{mnt_point}' partition filesystem is '$_->{fs_type}'");
```

Comment:
The 1st parameter in `record_info` just admit a few characters, but in this case does not differentiate between fs and size at first view, you use for example "Check size /" "Check fs_type /" to have better visualization.

Reply:
Done